bitswap/README.md

```diff
@@ -1,120 +1,12 @@
-go-bitswap
-==================
+Bitswap (moved to `blockexchage` and `swap`)
```

Comment:
:thought_balloon:  This readme will be what current bitswap users see, so good opportunity to point people at examples that use new API:
- https://github.com/ipfs/boxo/tree/main/examples/bitswap-transfer 
- https://github.com/ipfs/rainbow/blob/main/setup_bitswap.go (once we refactor both)